### PR TITLE
MSI: add single-cluster sanity tests 

### PR DESCRIPTION
**Pull request by rnetser**
Requires https://github.com/RedHatQE/managed-services-integration-tests/pull/531

### CODE REVIEW

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-single-cluster-smoke-v4.14-singlecluster-rosa-4-14-candidate-smoke

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-single-cluster-smoke-v4.14-singlecluster-rosa-4-14-candidate-smoke

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-single-cluster-smoke__v4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      HOSTED_CP: "false"
+      MULTI_AZ: "false"
+      OPENSHIFT_VERSION: "4.14"
+      READY_WAIT_TIMEOUT: 60m
+      REGION: us-west-2
+      REPLICAS: "2"
+      ROSA_LOGIN_ENV: staging
+    test:
+    - ref: cluster-msi-cluster-sanity
+    workflow: rosa-aws-sts
```
Comment (omrirh) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-single-cluster-smoke__v4.14.yaml:36]: why not using openshift-cli-installer ?

Comment (rnetser) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-single-cluster-smoke__v4.14.yaml:36]: this is for generic cluster sanity; not deplyed using `openshift-cli-installer`; will update the documntation

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-single-cluster-smoke-v4.14-singlecluster-rosa-4-14-candidate-smoke

**Pull request merged**